src/settings.rs

```diff
@@ -27,6 +27,14 @@ impl RuleStrategy {
         false
     }
 }
+impl Default for RuleStrategy {
+    fn default() -> Self {
+        RuleStrategy {
+            rule: "RunAsAny".to_string(),
+            ranges: vec![],
+        }
+    }
```

Comment:
Cannot help but think `RuleStrategy` should be an enum, where the `MustRunAs` variant is the one containing the list of ranges.

This would allow us to not build incorrect settings like:

```
rule: RunAsAny
ranges:
  - min: 1000
    max: 2000
```

Reply:
Agree with this, but I'm happy punting it forward for now.

Reply:
Yeah, I agree as well. @viccuad I think you can convert this comment in an issue and move on if you do not want to fix that now. Thus, someone can do that in the future.

Reply:
Done in https://github.com/kubewarden/user-group-psp-policy/issues/16! thanks both for the reviews!